packages/backend/src/connector/index.ts

```diff
@@ -1,4 +1,4 @@
-import IORedis from 'ioredis'
+import Redis from 'ioredis'
```

Comment:
There's a few other `import IORedis from 'ioredis'` if we want to change them all to be consistent
https://github.com/interledger/rafiki/search?q=IORedis

Reply:
Nice catch!